src/lxc/lxc_execute.c

```diff
@@ -139,6 +145,12 @@ int main(int argc, char *argv[])
 	if (lxc_config_define_load(&defines, conf))
 		return 1;
 
+	if (my_args.uid)
+		conf->init_uid = my_args.uid;
```

Comment:
Passing these as additions the existing `conf` seemed like the most trivial way to implement this. If you would prefer that they get passed explicitly as args it will require updating a number of call sites. I don't feel strongly at all and will defer to you. If we do decide to go with using `conf`, I could update [lxc_conf_init](https://github.com/ptoomey3/lxc/blob/1f885925ced89360bb84f199632d7101e6f7e469/src/lxc/conf.c#L2553) to default these values to 0.